### PR TITLE
to show full readme.rst in pypi

### PR DESCRIPTION
**Pull request by juanpabloaj**

### CODE REVIEW

**Comment by michael-lazar** [comment]
Thanks for contributing! I tried putting this up at https://testpypi.python.org/pypi/rtv.
Looks good except for the broken image link. Do you think you can fix that?

File: setup.py [hunk 1]
```diff
@@ -2,16 +2,17 @@
 #python setup.py develop --user
 #python setup.py develop --user --uninstall
 
+
+def read(*paths):
+    """ read files """
+    with open(os.path.join(*paths), 'r') as filename:
```
Comment (michael-lazar) [inline setup.py]: Missing 
```python
import os
```

**Comment by juanpabloaj** [comment]
sorry, now I added the

     import os

**Comment by juanpabloaj** [comment]
why don't allow the image in a external service ? like imgur 

I make this with my gif in the readme.rst 

https://pypi.python.org/pypi/pip-init
https://raw.githubusercontent.com/juanpabloaj/pip-init/master/README.rst

**Comment by michael-lazar** [comment]
On one hand it's nice to have the image in the project's version control so we can track the changes. On the other hand putting it on imgur makes it easier for people to share.

Here's a link http://i.imgur.com/4a3Yrov.gif
Can you update to README to point to that? I'll merge this when I get home.

**Comment by juanpabloaj** [comment]
ok,  the image was updated ;)

**Pull request merged**